### PR TITLE
fix: tekton operator overwrites configs

### PR DESCRIPTION
**Pull request by yftacherzog**
The tekton operator seems to overwrite whatever tekton configs that are in place when it is deployed. For that reason, we should wait for those configs to appear before deploying our own configs.

### CODE REVIEW

File: deploy-deps.sh [hunk 1]
```diff
@@ -48,13 +48,15 @@ deploy_tekton() {
     # Operator
     kubectl apply -k "${script_path}/dependencies/tekton-operator"
     retry kubectl wait --for=condition=Ready -l app=tekton-operator -n tekton-operator pod --timeout=240s
+    # Wait for the operator to create configs for the first time before applying configs
+    kubectl wait --for=condition=Ready TektonConfig/config --timeout=360s
```
Comment (avi-biton) [inline deploy-deps.sh]: I tested it with `240s` for several times and it worked.
Do we want to wait this long?

Comment (yftacherzog) [reply deploy-deps.sh]: Just because that was the number on the original wait line. It might be that @gbenhaim had some tries in which 240 was not enough.

Comment (avi-biton) [inline deploy-deps.sh]:
```suggestion
    kubectl wait --for=condition=Ready tektonconfig/config --timeout=360s
```
for consistency with the second call

Comment (yftacherzog) [reply deploy-deps.sh]: done

**Review by gbenhaim — APPROVED** [review]

**Pull request merged**